app/views/pages/feedback.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= govuk_link_to("Home", "/") %>
+<%= govuk_label text: 'Give feedback on Request personal information', tag: 'h1', size: 'm' %>
+<%= govuk_label text: 'Satisfaction survey', tag: 'h2', size: 'm' %>
```

Comment:
These shouldn't be labels, just an H1 and H2.

Reply:
Look at the rest of the application to see how this is done. (you need to add the expected class)